p256-cm4/Cargo.toml

```diff
@@ -16,8 +16,13 @@ all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
+default = [ "p256", "signature" ]
```

Comment:
Generally I prefer having no default features for embedded libraries, but I don't feel too strongly about this.